pkg/genesis/vesting_accounts.go

```diff
@@ -0,0 +1,155 @@
+package genesis
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/celestiaorg/celestia-app/app/encoding"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	vestingtypes "github.com/cosmos/cosmos-sdk/x/auth/vesting/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+)
+
+func NewGenesisRegularAccount(
+	address string,
+	balances sdk.Coins,
+) (account authtypes.GenesisAccount, balance banktypes.Balance, err error) {
+	sdkAddr, err := sdk.AccAddressFromBech32(address)
+	if err != nil {
+		return account, balance, err
+	}
+
+	balance = banktypes.Balance{
+		Address: address,
+		Coins:   balances,
+	}
+	bAccount := authtypes.NewBaseAccountWithAddress(sdkAddr)
+
+	return authtypes.GenesisAccount(bAccount), balance, nil
+}
+
+// NewGenesisDelayedVestingAccount creates a new DelayedVestingAccount with the specified parameters.
+// then returns the created account converted to genesis account type and the account balance
```

Comment:
[nits]
- capitalize first word in sentence
- period at end of sentence
- rewrap

```suggestion
// NewGenesisDelayedVestingAccount creates a new DelayedVestingAccount with the
// specified parameters. It returns the created account converted to genesis
// account type and the account balance.
```

I'm starting to think the comment rewrapping thing to 80 character width is a waste of dev productivity b/c we commonly have nits about it. Since it isn't enforced by a linter, I don't think it is worth doing.

Reply:
[no change needed] I'm not sure why the delayed vesting account constructor is the only function in this file that has a godoc. It seems comparable to `NewGenesisPeriodicVestingAccount` and `NewGenesisContinuousVestingAccount`.